src/TransactionTable.ts

```diff
@@ -0,0 +1,93 @@
+// Global to preserve active row after editing a cell in it via keyboard
+let lastActiveRowIndex = "";
+
+function isSkippableRow(
+  row: Element | null | undefined
+): row is HTMLTableRowElement {
+  // This is the "Show/Hide pending transactions" row, skip it
+  return (
+    !!row && row.tagName === "TR" && !row.classList.contains("transaction-row")
```

Comment:
I believe this breaks navigation on the recurring/suggested page. It may be better to find a class/attribute of the pending transactions row than to look for a specific attribute of only the transactions table.

Reply:
I've been pretty busy, so for the sake of getting this into people's hands—I'm merging this and then we can open an issue to fix if it's a problem.